gcp_flowlogs_reader/gcp_flowlogs_reader.py

```diff
@@ -229,7 +229,7 @@ def _reader(self):
                 iterator = self.logging_client.list_entries(
                     filter_=expression,
                     page_size=self.page_size,
-                    projects=[project],  # only collects current project flows
+                    resource_names=[project],  # only current project flows
                 )
                 for page in iterator.pages:
```

Comment:
Based on a field test, it looks like `list_entries` returns some sort of paging generator now. The returned object doesn't have a `pages` attribute, but can be iterated directly.

i.e. dop this loop and make it `for flow_entry in iterator`.